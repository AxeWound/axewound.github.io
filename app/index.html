<!DOCTYPE html>
<html >
<head>
	<title>Title of the document</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<link rel="stylesheet" type="text/css" href="style-hamburg.css"/>
	<link rel="stylesheet" type="text/css" href="hamburgers.css">

	<script src="https://kit.fontawesome.com/684992f13d.js" crossorigin="anonymous"></script>
	<link href="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.9.0/slick.min.css" rel="stylesheet"/>
	<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
	<script src="https://cdnjs.cloudflare.com/ajax/libs/slick-carousel/1.9.0/slick.min.js"></script>
	<link rel="stylesheet" href="style.css">
	
	

</head>
<body>

	



	<header>
		<div class="container">

			<div class="logo">
				<img src="img/logo.svg">
			</div>
			<div class="header_menu">
				<nav>
					<ul>
						<li><a href="">ГЛАВНАЯ</a></li>
						<li><a href="">ГАЛЕРЕЯ</a></li>
						<li><a href="">ПРОЕКТЫ</a></li>
						<li><a href="">СЕРТИФИКАТЫ</a></li>
						<li><a href="">КОНТАКТЫ</a></li>
					</ul>
				</nav>
			</div>
			<div class="top_line">
				<div class="logo">
					<img src="img/logo.svg">
				</div>
				<div class="header-hambureger">

					<div class="hamburger hamburger--spring">
						<div class="hamburger-box">
							<div class="hamburger-inner"></div>
						</div>
					</div>
				</div>
				<div class="menu">
					<div class="header_menu">
						<nav>
							<ul>
								<li><a href="">ГЛАВНАЯ</a></li>
								<li><a href="">ГАЛЕРЕЯ</a></li>
								<li><a href="">ПРОЕКТЫ</a></li>
								<li><a href="">СЕРТИФИКАТЫ</a></li>
								<li><a href="">КОНТАКТЫ</a></li>
							</ul>
						</nav>
					</div>
				</div> 

			</div>

		</header>
		<section class="section_first">
			<div class="container">
				<div id="slider slider-section">
					<div class="first_block_slider">
						<div class="name_block">
							<h1>PROJECT <br><span>HOME</span></h1>
							<div class="button">

								<input id="btn-prev" type="image" src="img/arrow-left.svg">

								<input id="btn-next" type="image" src="img/1920px-Feather-arrows-arrow-right.svg.png">
							</div>
							<div class="number_slider">
								<span class="pagingInfo"></span>


							</div>
						</div>
						<div class="first_block_images">

							<div class="images slideshow">
								<img src="img/house_img.png"  class="active">

								<img src="img/building.jpeg ">
							</div>
							<button class="button_all"><span>ВЗГЛЯНУТЬ</span> <i class="fas fa-long-arrow-alt-right"></i></button>
						</div>

					</div>
				</div>
			</div>

		</section>

		<section class="section_second">
			<div class="container">
				<div class="images_second">
					<div class="img_first">
						<img src="img/Rectangle 2.svg">
						<img src="img/Rectangle 1.svg">
					</div>
					<div class="img_second">
						<img src="img/Rectangle 3.svg">
					</div>
				</div>
				<div class="company">
					<h2>О компании</h2>
					<p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.</p>
					<button class="button_all"><span>ЧИТАТЬ</span> <img src="img/arrow_right.svg"></button>
				</div>
			</div>
		</section>
		<section class="section_third">
			<div class="container">
				<h2>Основные задачи</h2>
				<div class="third_block">
					<div>
						<h3>1</h3>
						<p>Создание комфортных условий и повышение качества обслуживания клиентов</p>
					</div>
					<div>
						<h3>2</h3>
						<p>Постоянно совершенствовать качество предоставляемых услуг путем обучения персонала, закупки нового оборудования и усиленной рекламы на рынке</p>
					</div>
				</div>
			</div>
		</section>
		<section class="section_fourth">
			<div class="container">
				<h2>Наши проекты</h2>
				<div class="block_four_img">
					<div class="line_img">
						<div class="four_centr">
							<h4>ДОСУГОВЫЙ ЦЕНТР</h4>
							<a href="">ПОДРОБНЕЕ</a>
							<i class="fas fa-long-arrow-alt-right"></i>
							</div>
							<div>
								<img src="img/image15.svg">
							</div>
						</div>
						<div class="line_img_second">
							<img src="img/image 16.svg">
							<img src="img/image 17.svg">
							<img src="img/image 18.svg">
						</div>
					</div>
					<button class="button_all"><span>ВЗГЛЯНУТЬ</span> <i class="fas fa-long-arrow-alt-right"></i></button>
				</div>
			</section>
			<section class="section_fifth">
				<div class="container">
					<h2>Связаться с нами</h2>
					<div class="regist">
						<div class=" registration_form ">
							<form name="RegForm" id="signup" class="form"> 
								<div class="form-group">
									<div class="form-field">

										<input id="username" name="name"  type="text" class="form-control" id="formGroupExampleInput" placeholder="Имя">
									</div>
									<div class="form-field">
										<input class="form-control" id="phone" type="tel" placeholder="Номер телефона*">
									</div>
									<div class="form-field">

										<input  id="email" type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="E-mail*">
									</div>
									<div class="form-field">
										<input id="username" name="int"  type="text" class="form-control" id="formGroupExampleInput" placeholder="Интересующий товар/услуга">
									</div>
									<div class="form-field">
										<textarea placeholder="Сообщение*" rows="10" cols="45" name="text"></textarea>
									</div>
								</div>
							</form>
							<div class="form-check">
								<span>Отправляя заявку Вы соглашаетесь
									с политикой 
								конфиденциальности</span>
								<button class="button_all"><span>ОТПРАВИТЬ</span><i class="fas fa-long-arrow-alt-right"></i></button>
							</div>
						</div>
						<img src="img/image 12.svg">
					</div>

				</div>
			</section>
			<footer>
				<div class="container">
					<div class="footer_section">
						<div class="footer_logo">
							<img src="img/footer_logo.svg">
						</div>
						<div class="footer_menu">
							<nav>
								<span>Информация</span>
								<ul>
									<li><a href="">Главная</a></li>
									<li><a href="">Галерея</a></li>
									<li><a href="">Проекты</a></li>
									<li><a href="">Сертификаты</a></li>
									<li><a href="">Контакты</a></li>
								</ul>
							</nav>
						</div>
						<div class="footer_menu address">
							<nav>
								<span>Контакты</span>
								<ul>
									<li><a href=""><img src="img/Vector.svg">100000, Республика  Казахстан, 
									г. Караганда, ул. Телевизионная 10</a></li>
									<li><a href=""><img src="img/Vector-1.svg">+38 (000) 000 00 00</a></li>
									<li><a href=""><img src="img/Group.svg">mail@gmail.com</a></li>
								</ul>
							</nav>
						</div>
						<div class="icon footer_menu">
							<nav>
								<span>Социальные сети</span>
								<ul>
									<li><a href=""><img src="img/facebook.svg"></a></li>
									<li><a href=""><img src="img/Shape.svg"></a></li>
									<li><a href=""><img src="img/Shape-2.svg"></a></li>
									<li><a href=""><img src="img/Shape-1.svg"></a></li>
								</ul>
							</nav>
						</div>
					</div>
				</div>
			</footer>
			<hr>
			<footer class="footer">
				<div class="container">
					<h6>© 2019 Digital Project. Все права защищены.</h6>
				</div>
			</footer>
		</body>
		<script>
			var forEach=function(t,o,r){if("[object Object]"===Object.prototype.toString.call(t))for(var c in t)Object.prototype.hasOwnProperty.call(t,c)&&o.call(r,t[c],c,t);else for(var e=0,l=t.length;l>e;e++)o.call(r,t[e],e,t)};
			var hamburgers = document.querySelectorAll(".hamburger");
			if (hamburgers.length > 0) {
				forEach(hamburgers, function(hamburger) {
					hamburger.addEventListener("click", function() {
						this.classList.toggle("is-active");
					}, false);
				});
			}
		</script>
		<script>
			$( document ).ready(function() {
				$( ".cross" ).hide();
				$( ".menu" ).hide();
				$( ".hamburger" ).click(function() {
					$( ".menu" ).slideToggle( "slow", function() {
						$( ".cross" ).show();
					});
				});
			});
		</script>
		<script src="script.js"></script>
		</html>