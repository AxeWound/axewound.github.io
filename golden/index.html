<!DOCTYPE html>
<html lang="en">
<head>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/style.css">
	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
</head>
<body>
<header class="header">
	<div class="container">
<nav class="navbar navbar-expand-lg navbar-light ">
		    <div class="d-flex flex-grow-1">
		        <span class="w-100 d-lg-none d-block"></span>
		        <a class="navbar-brand d-none d-lg-inline-block" href="#">
		           	<h1 class="logo wow bounceInLeft"><img src="img/logo.png"></h1>
		        </a>
		        <a class="navbar-brand-two mx-auto d-lg-none d-inline-block" href="#">
		            <h1 class="logo"><img src="img/logo.png"></h1>
		        </a>
		        <div class="w-100 text-right menu-nav-right">
		            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#myNavbar">
		            <span class="navbar-toggler-icon"></span>
		            </button>
		        </div>
		    </div>
		    <div class="collapse navbar-collapse flex-grow-1 text-right mt-2 wow bounceInRight" id="myNavbar">
		        <ul class="navbar-nav flex-nowrap menu-right-right">
		            <li class="nav-item">
		                <a href="#" class="nav-link menu-item nav-active">Home</a>
		            </li>
		            <li class="nav-item">
		                <a href="#Pages" class="nav-link menu-item ">About</a>
		            </li>
		            
		            <li class="nav-item">
		                <a href="#Features"  class="nav-link menu-item">Portfolios</a>
		            </li>
		            <li class="nav-item">
		                <a href="#Extensions" class="nav-link menu-item">Team</a>
		            </li>
		            <li class="nav-item">
		                <a href="#Tutorials" class="nav-link menu-item">Blog</a>
		            </li> 
		             <li class="nav-item">
		                <a href="#Contact" class="nav-link menu-item">Contact</a>
		            </li> 
		        </ul>
		    </div>
		</nav>
</div>
</header>
<section class="section_first">
	<div class="container">
		<div class="row ">
			<div class="col  text-center text-lg-left">
				<h2>Grab Your Copy Of</h2>
				<h1>The <span>Premium</span> Quality<br>PSD Template For <span>free</span></h1>
				<button class="section_first_button btn btn-primary btn-lg " type="submit">Download<i class="fas fa-download"></i></button>
			</div>
		</div>
	</div>
</section>
<section class="section_second">
	<div class="container">
		<div class="row">
			<div class="col">
			<h3>Know About Us</h3>
			<hr class="all-hr">
			<p class="col-6 all_p">Nam liber tempor cum soluta nobis eleifend option congue nihil imperdiet doming id quod mazim </p>
		</div>
		</div>
	</div>
</section>
<section class="section_three">
		<div class="row">
			<div class="col-6 section_three_left">
				<div class="section_three_textInfo_left">
				<h4>We Are the best</h4>
				<p>Consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat <br><br>
				Volutpat wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo consequat. Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Nam liber tempor cum </p>
				<button type="button" class="btn btn-outline-dark">Read more<i class="fas fa-long-arrow-alt-right"></i></button>
			</div>
			</div>
			<div class="col-6 section_three_right"></div>
		</div>
</section>
<section class="section_four">
	<div class="container">
		<div class="row">
			<div class="col">
				<h4>Services We provide</h4>
				<p class="col-11 all_p">Volutpat wisi enim ad minim veniam, quis nostrud exerci tation ullamcorper suscipit lobortis nisl ut aliquip ex ea commodo 
				consequat. Duis autem veleum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu 
				feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent </p>
			</div>
		</div>
		<div class="section_four_block row">
				<div class="section_four_box col-4">
					<img src="img/icon/1.png">
					<h5>ConsulTancy</h5>
					<p>Claritas est etiam processus dynaicus qui sequitur mutationem consuetudium lectorum. Mirum est notare quam </p>
				</div>
				<div class="section_four_box col-4">
					<img src="img/icon/2.png">
					<h5>ConsulTancy</h5>
					<p>Claritas est etiam processus dynaicus qui sequitur mutationem consuetudium lectorum. Mirum est notare quam </p>
				</div>
				<div class="section_four_box col-4">
					<img src="img/icon/3.png">
					<h5>ConsulTancy</h5>
					<p>Claritas est etiam processus dynaicus qui sequitur mutationem consuetudium lectorum. Mirum est notare quam </p>
				</div>
		</div>
	</div>
</section>

<section class="section-five">
	<div class="container">
		<h4>Our Skills</h4>
		<div class="row">
			<div class="col section-five-box">
				<img src="img/single-skill copy 2.png">
				<h5>Marketing</h5>
			</div>
			<div class="col section-five-box">
				<img src="img/single-skill copy 2.png">
				<h5>Marketing</h5>
			</div>
			<div class="col section-five-box">
				<img src="img/single-skill copy 2.png">
				<h5>Marketing</h5>
			</div>
			<div class="col section-five-box">
				<img src="img/single-skill copy 2.png">
				<h5>Marketing</h5>
			</div>
			<div class="col section-five-box">
				<img src="img/single-skill copy 2.png">
				<h5>Marketing</h5>
			</div>

		</div>
	</div>
</section>
<section class="section-six">
	<div class="container">
		<div class="row">
			<div class="col"></div>
			<div class="col-6 justify-content-center text-center">
				<h3>Some Recent Projects</h3>
				<hr class="all-hr">
				<p class="text-center ">Nam liber tempor cum soluta nobis eleifend option congue nihil imperdiet doming id quod mazim </p>
			</div>
			<div class="col"></div>
		</div>
	</div>
</section>
<section class="section-seven">
	<div class="section-seven-image">
		<img src="img/Rectangle56.png">
		<img src="img/Rectangle562.png">
		<img src="img/Rectangle563.png">
		<img src="img/Rectangle564.png">
	</div>
	<div class="section-seven-box-center">
		<div class="container">
			<div class="row">
				<div class="col"></div>
				<div class="col-8  fixed-center">
					<h4>LATEST FROM TWITTER</h4>
					<p>Trying out velositey - a simple website building extension for Photoshop.<br><br>
					<a href="#">buff.ly/NC8gog</a></p>
				</div>
				<div class="col"></div>
			</div>
		</div>
	</div>
	<div class="section-seven-image">
		<img src="img/Rectangle56.png">
		<img src="img/Rectangle562.png">
		<img src="img/Rectangle563.png">
		<img src="img/Rectangle564.png">
	</div>
	<div class="container">
		<div class="section-seven-btn row">
			<div class="col">
				<button type="button" class=" btn btn-outline-dark">See all<i class="fas fa-long-arrow-alt-right"></i></button>
			</div>
		</div>
	</div>
</section>
<section class="section-eight">
	<div class="container">
		<div class="row">
			<div class="col"></div>
			<div class="col-6 fixed-center">
				<h3>Our Team Members</h3>
				<hr class="all-hr">
				<p>Nam liber tempor cum soluta nobis eleifend option congue nihil imperdiet doming id quod mazim </p>
			</div>
			<div class="col"></div>
		</div>
		<div class="row section-eight-information">
			<div class="col-4 section-eight-people">
				<img src="http://placehold.it/367x370" >
					<div class="socialGroup ">
						<div class="row">
							<div class="col"></div>
							<div class="col-3 section-eight-people-mini ">
								 <a href="#"><img src="img/faceSlayer/fb.png"></a>
							</div>
							<div class="col-3 section-eight-people-mini">
								  <a href="#"><img src="img/faceSlayer/tw.png"></a>
							</div>
							<div class="col-3 section-eight-people-mini">
								<a href="#"><img src="img/faceSlayer/in.png"></a>
							</div>
							<div class="col"></div>
						</div>
				    </div>
				
				<div class="section-eight-description mt-4">
					<h6 class="my-4">Mitchell Anderson</h6>
		    		<h5 class="my-3">Creative Director</h5>
				</div>
			</div>
			<div class="col-4 section-eight-people">
				<img src="http://placehold.it/367x370">
					<div class="socialGroup">
					    <div class="row">
						<div class="col"></div>
							<div class="col-3 section-eight-people-mini">
								 <a href="#"><img src="img/faceSlayer/fb.png"></a>
							</div>
							<div class="col-3 section-eight-people-mini">
								  <a href="#"><img src="img/faceSlayer/tw.png"></a>
							</div>
							<div class="col-3 section-eight-people-mini">
								<a href="#"><img src="img/faceSlayer/in.png"></a>
							</div>
							<div class="col"></div>
						</div>
				    </div>
				
				<div class="section-eight-description mt-4">
					<h6 class="my-4">Mitchell Anderson</h6>
		    		<h5 class="my-3">Creative Director</h5>
				</div>
			</div>
			<div class="col-4 section-eight-people">
				<img src="http://placehold.it/367x370">
					<div class="socialGroup">
					   <div class="row ">
							<div class="col"></div>
							<div class="col-3 section-eight-people-mini">
								 <a href="#"><img src="img/faceSlayer/fb.png"></a>
							</div>
							<div class="col-3 section-eight-people-mini">
								  <a href="#"><img src="img/faceSlayer/tw.png"></a>
							</div>
							<div class="col-3 section-eight-people-mini">
								<a href="#"><img src="img/faceSlayer/in.png"></a>
							</div>
							<div class="col"></div>
						</div>
				    </div>
				
				<div class="section-eight-description mt-4">
					<h6 class="my-4">Mitchell Anderson</h6>
		    		<h5 class="my-3">Creative Director</h5>
				</div>
			</div>
		</div>
	</div>
</section>
<section class="sectionEight">
	<div class="container">
		<div class="row">
			<div class="col">
				<div class="slideshow-container">
					<h4>What Our Clients are saying</h4>
					<div class="mySlides fade">
		    			<div class="reviewsAll">
							<div class="reviews">
								<p>“It was a pleasure to work with Imran. He is very dedicated and professional. He worked very hard to satisfy our requirements and the communication was great.”</p>
								<h2>John Doe <br><br>
								<font>www.yourwebsite.zt</font></h2>
							</div>
						</div>
					</div>
					<div class="mySlides fade">
			    		<div class="reviewsAll">
							<div class="reviews">
								<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Perspiciatis quia quas, quis illo adipisci voluptate ex harum iste commodi nulla dolor.</p>
								<h2>Alex Drogba <br><br>
								<font>www.yourwebsite.zt</font></h2>
							</div>
						</div>
					</div>
					<div class="mySlides fade">
						<div class="reviewsAll">
							<div class="reviews">
								<p>Once upon a time all the Rivers combined to protest against the action of the Sea in making their waters salt. “When we come to you,” said they to the Sea.</p>
								<h2>Joe Hurt <br><br>
								<font>www.yourwebsite.zt</font></h2>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
		<div style="text-align:center">
			<span class="dot" onclick="currentSlide(1)"></span> 
		  	<span class="dot" onclick="currentSlide(2)"></span> 
		  	<span class="dot" onclick="currentSlide(3)"></span> 
		</div>
	</div>		
</section>
<section class="sectionNine">
<div class="container">
	<div class="row">
		<div class="col-12 fixed-center">
			<h3>Our Blog</h3>
			<hr class="all-hr">
			<div class="col">
			<p>Nam liber tempor cum soluta nobis eleifend option congue nihil imperdiet doming id quod mazim </p></div>
		</div>
		<div class="col-6 classl-left sectionNine-classl-left">
			<div class="circle"></div>
			<div class="sectionNine-image">
				<img class="mySlides3"  src="img/image.png">
				<img class="mySlides3"  src="img/image1.png">
				<img class="mySlides3"  src="img/image2.png">
				<a class="w3-button w3-display-left" onclick="plusDivs3(-1)"><img src="img/fromleft.png"></a>
				<a class="w3-button w3-display-right" onclick="plusDivs3(+1)"><img src="img/fromright.png"></a>
			</div>
			<div class="circle-two"></div>
			<div class="sectionNine-image-right">
				<figcaption>20 March 2014</figcaption>
				<h5 class="">Typi non habent claritatem insitam</h5>
				<p>Duis autem vel eum iriure dolor in hendrerit in vulputate velit<br> esse molestie consequat, vel illum dolore eu feugiat nulla<br> facilisis at vero eros et accumsan et iusto odio dignissim qui <span>[...]</span></p>
			</div>
			<div class="circle-three"></div>
			<div class="sectionNine-image">
				<img class="mySlides1"  src="img/image.png">
				<img class="mySlides1"  src="img/image1.png">
				<img class="mySlides1"  src="img/image2.png">
				<a class="w3-button w3-display-left" onclick="plusDivs1(-1)"><img src="img/fromleft.png"></a>
				<a class="w3-button w3-display-right" onclick="plusDivs1(+1)"><img src="img/fromright.png"></a>
			</div>
		</div>
		<div class="col-6 classl-left sectionNine-classl2">
			<figcaption>20 March 2014</figcaption>
			<h5 class="">Typi non habent claritatem insitam</h5>
			<p>Duis autem vel eum iriure dolor in hendrerit in vulputate velit<br> esse molestie consequat, vel illum dolore eu feugiat nulla<br> facilisis at vero eros et accumsan et iusto odio dignissim qui <span>[...]</span></p>
			<div class="sectionNine-image">
				<img class="mySlides2"  src="img/image.png">
				<img class="mySlides2"  src="img/image1.png">
				<img class="mySlides"  src="img/image2.png">
				<a class="w3-button w3-display-left" onclick="plusDivs2(-1)"><img src="img/fromleft.png"></a>
				<a class="w3-button w3-display-right" onclick="plusDivs2(+1)"><img src="img/fromright.png"></a>
			</div>
			<figcaption>20 March 2014</figcaption>
			<h5 class="">Typi non habent claritatem insitam</h5>
			<p>Duis autem vel eum iriure dolor in hendrerit in vulputate velit<br> esse molestie consequat, vel illum dolore eu feugiat nulla<br> facilisis at vero eros et accumsan et iusto odio dignissim qui <span>[...]</span></p>
		</div>
		<div class="col-12">
		<div class="btn-all-post ">
			<a href="#"><img src="img/btn-all-post.png"></a>
		</div>
		</div>
	</div>
</div>
</section>

<section class="section-eleven">
	<div class="container">
		<div class="row">
			<div class="col "">
				<h3 class="mb-5">Contact</h3>
				<form>
					<div class="form-row ">
					    <div class="col d-flex justify-content-end mr-3">
					      	<input type="text" class="form-control section-eleven-form" placeholder="Name">
					    </div>
					    <div class="col ml-3">
					      	<input type="email" class="form-control section-eleven-form" id="inputEmail4" placeholder="Email">
					    </div>
					</div>
					<div class="col section-eleven-form-fix d-flex justify-content-center mt-4 ">
					    <textarea class="form-control section-eleven-textarea" id="exampleFormControlTextarea1" rows="5" placeholder="Message..."></textarea>
					</div>
				</form>
				<div class="row d-flex justify-content-center">
					<button type="button" class="btn btn-outline-dark mt-5">Submit</button>
				</div>
			</div>
		</div>
	</div>
</section>
<footer class="footer">
	<div class="container">
		<div class="row">
			<div class="col">
				<img src="img/Mairala.png" class="my-5">
				<p>Duis autem vel eum iriure dolor in hen-drerit in vulputate velit esse molestie consequat, vel illum dolore.
				<br><br>
				Eugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Nam liber tempor cum soluta </p>
			</div>
			<div class="col">
				<div class="footer-box my-5" >
					<div class="timeData">
						<h5 >Recent Posts</h5>
						<h6 >March 30, 2014</h6><span>Duis autem vel eum iriure dolor</span>
						<hr>
					</div>
					<div class="timeData">
						<h6 class="mt-4">March 30, 2014</h6><span>Duis autem vel eum iriure dolor</span>
						<hr>
					</div>
					<div class="timeData">
						<h6 class="mt-4">March 30, 2014</h6><span>Duis autem vel eum iriure dolor</span>
						<hr>
					</div>
				</div>
			</div>
			<div class="col">
				<div class="footer-box my-5">
					<div class="timeData">
						<h5>Twiter Feeds</h5>	
						<h6><small>about 2 days ago</small> @kristianplatt  </h6><span>Good work buddy</span></h6>
						<hr>
					</div>
					<div class="timeData">
						<h6 class="mt-4"><small>about 2 days ago</small> @kristianplatt  </h6><span>Good work buddy</span>
						<hr>
					</div>
					<div class="timeData">
						<h6 class="mt-4"><small>about 2 days ago</small> @kristianplatt  </h6><span>Good work buddy</span></h6>
						<hr>
					</div>
				</div>
			</div>
			<div class="col">
				<div class="footer-box my-5">
					<h5 class="mb-5">Our Address</h5>
					<p>Investigationes demonstraverunt lectores legere me lius quod ii legunt saepius. Claritas est etiam processus dynamicus,luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Nam liber tempor cum soluta 
					<br>
					<br>
					<br>
					<i class="fas fa-map-marker-alt"></i>Moonshine St. 14/05 Light, Jupiter<br>
					<i class="fas fa-phone"></i>+00 (123) 456 78 90<br>
					<i class="far fa-envelope"></i>first.last@email.com</p>
				</div>
			</div>
		</div>
	</div>
	<div class="footer-footer">
		<div class="container">
			<div class="row">
				<div class="col text-center  ">
					<p class="my-4">©2014 Imran Khan All Rights Reserved
				</div>
			</div>
		</div>
	</div>
</footer>

<script src="js/js.js"></script>
<script src="js/js1.js"></script>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
</body>
</html>